packages/epo-react-lib/src/atomic/Carousel/Provider/index.tsx

```diff
@@ -3,7 +3,7 @@ import { CarouselConfig, CarouselOptions } from "@/types/carousel";
 import useCarousel from "@/hooks/useCarousel";
 import CarouselContext from "../context";
 
-interface CarouselProviderProps {
+export interface CarouselProviderProps {
```

Comment:
TypeScript throwing a warning that it couldn't get this interface for the definition files